commit 5649088bccc35f1a824e2b0c6482397b13c94abd
Author: Dave Anderson <anderson@redhat.com>
Date:   Thu May 5 09:01:35 2016 -0400

    Fix for a "[-Werror=misleading-indentation]" compiler warning that
    is generated by gdb-7.6/bfd/elf64-s390.c when building S390X in a
    Fedora Rawhide environment with gcc-6.0.0.
    (anderson@redhat.com)

diff --git a/gdb-7.6.patch b/gdb-7.6.patch
index 920ea1c..794555f 100644
--- a/gdb-7.6.patch
+++ b/gdb-7.6.patch
@@ -2206,3 +2206,20 @@ diff -up gdb-7.6/bfd/elf64-ppc.c.orig gdb-7.6/bfd/elf64-ppc.c
 +  gi->finished = 1; 
 +}
  #endif
+--- gdb-7.6/bfd/elf64-s390.c.orig
++++ gdb-7.6/bfd/elf64-s390.c
+@@ -323,10 +323,10 @@ elf_s390_reloc_name_lookup (bfd *abfd AT
+ 	&& strcasecmp (elf_howto_table[i].name, r_name) == 0)
+       return &elf_howto_table[i];
+ 
+-    if (strcasecmp (elf64_s390_vtinherit_howto.name, r_name) == 0)
+-      return &elf64_s390_vtinherit_howto;
+-    if (strcasecmp (elf64_s390_vtentry_howto.name, r_name) == 0)
+-      return &elf64_s390_vtentry_howto;
++  if (strcasecmp (elf64_s390_vtinherit_howto.name, r_name) == 0)
++    return &elf64_s390_vtinherit_howto;
++  if (strcasecmp (elf64_s390_vtentry_howto.name, r_name) == 0)
++    return &elf64_s390_vtentry_howto;
+ 
+   return NULL;
+ }
